src/vm/database/clarity_db.rs

```diff
@@ -106,6 +106,10 @@ pub trait BurnStateDB {
         height: u32,
         sortition_id: &SortitionId,
     ) -> Option<BurnchainHeaderHash>;
+    fn get_burn_header_hash_using_canonical_sortition(
```

Comment:
should this function be named something more like `get_burn_header_hash_using_height` instead?

Reply:
what were your thoughts here?

Reply:
oh whoops.. Well, the reason it is "using_canonical_sortition" is because the "canonical sortiiton" is the hidden argument. That is, the `height` part is in the signature. Compare with `get_burn_header_hash`. The idea is that these two functions are similar but this one uses an implicit sortition. `get_burn_header_hash` also doesn't mention height.

Reply:
ahh I see, sounds good to me.